lib/structures/Guild.js

```diff
@@ -119,9 +122,18 @@ class Guild extends Base {
                 }
             }
         }
+
+        this._fetchAllInvites();
         this.update(data);
     }
 
+    _fetchAllInvites() {
+        const botMember = this.members.get(this._client.user.id);
+        if(botMember && botMember.permission.has("manageGuild")) {
+            return this.getInvites().then((invites) => invites.forEach((invite) => this.invites.add(invite)));
```

Comment:
Also a bad idea, the library shouldn't make automatic REST calls that aren't strictly necessary